Code/MinimalLib/common.h

```diff
@@ -8,7 +8,9 @@
 //  of the RDKit source tree.
 //
 #pragma once
+#ifdef __GNUC__
```

Comment:
If this is going to stay here, and I don't think it needs to after these changes are made, we probably also want it one for clang, right?

Reply:
You added that pragma on 21/12/2023, I think to make CI happy. I only added the `#ifdef` guard to silence the "unknown pragma" warning in emscripten builds. It ended up in this PR but is not really connected to the fingerprint work.